tests/test_backend_rest.py

```diff
@@ -0,0 +1,34 @@
+from types import MethodType
+
+from bugzilla._backendrest import _BackendREST
+from bugzilla._session import _BugzillaSession
+
+
+def test_getbug():
+    backend = _BackendREST(url="http://example.com",
+                           bugzillasession=_BugzillaSession(url="http://example.com",
+                                                            user_agent="py-bugzilla-test",
+                                                            sslverify=False,
+                                                            cert=None,
+                                                            tokencache={},
+                                                            api_key="",
+                                                            is_redhat_bugzilla=False))
+
+    def _assertion(self, *args, **kwargs):
+        self.assertion_called = True
+        assert args and args[0] == url
+
+    backend._get = MethodType(_assertion, backend)
+
+    for _ids, aliases, url in (
+            (1, None, "/bug/1"),
+            ([1], [], "/bug/1"),
+            (None, "CVE-1999-0001", "/bug/CVE-1999-0001"),
+            ([], ["CVE-1999-0001"], "/bug/CVE-1999-0001"),
+            (1, "CVE-1999-0001", "/bug"),
+    ):
+        backend.assertion_called = False
+
+        backend.bug_get(_ids, aliases, {})
+
+        assert backend.assertion_called is True
```

Comment:
`./setup.py pylint` throws some errors. I'm open to modernizing the formatting suggestions here or using tools like pre-commit and/or black, but let's keep this clean until then.

```
tests/test_backend_rest.py:9: [E501] line too long (85 > 79 characters)
tests/test_backend_rest.py:10: [E501] line too long (90 > 79 characters)
tests/test_backend_rest.py:15: [E501] line too long (86 > 79 characters)
running pylint-3
************* Module tests.test_backend_rest
tests/test_backend_rest.py:17:0: W0613: Unused argument 'kwargs' (unused-argument)
tests/test_backend_rest.py:21:4: W0212: Access to a protected member _get of a client class (protected-access)
```

Reply:
Oh! I'm so used to pylint being part of a CI pipeline that I did not check. I have changed the code.

> I'm open to modernizing the formatting suggestions

I created a proposal in #185 :slightly_smiling_face: